internal/view/container.go

```diff
@@ -163,6 +166,34 @@ func (c *Container) attachCmd(evt *tcell.EventKey) *tcell.EventKey {
 	return nil
 }
 
+func (c *Container) copyImageToClipboard(evt *tcell.EventKey) *tcell.EventKey {
+	path := c.GetTable().GetSelectedItem()
+	if path == "" {
+		c.App().Flash().Err(errors.New("could not retrieve the selected item"))
+		return evt
+	}
+
+	po, err := fetchPod(c.App().factory, c.GetTable().Path)
+	if err != nil {
+		c.App().Flash().Err(err)
+		return evt
```

Comment:
We are handling this action. Tho we are erroring out, best not to propagate by just returning nil